flake.nix

```diff
@@ -12,11 +12,22 @@
           inherit system;
           overlays = [ poetry2nix.overlay ];
         };
+        poetryOverrides = self: super: {
+          typing-extensions = super.typing-extensions.overridePythonAttrs (
```

Comment:
Refs https://github.com/nix-community/poetry2nix/issues/218#issuecomment-981615612

Reply:
Let's add that as a comment